src/main/scala/dameo/variant/Dameo.scala

```diff
@@ -19,9 +19,4 @@ case object Dameo
   def perfId: Int    = 800
 
   override def baseVariant: Boolean = true
-
-  // TODO Dameo set this
-  override def initialFen = format.FEN("")
```

Comment:
I know its technically unnecessary, but can you put the Dameo starting FEN in here as well? It makes checking things easier, and will help with any future changes if we do go down the route of migrating more draughts variants into here